src/picongpu/include/particles/particleToGrid/derivedAttributes/LarmorPower.hpp

```diff
@@ -55,16 +55,19 @@ namespace derivedAttributes
         const float_X gamma2 = gamma * gamma;
         const float_X c2 = SPEED_OF_LIGHT * SPEED_OF_LIGHT;
 
-        const float3_X mom_dt = mom - mom_mt1;
+        const float3_X mom_dt = (mom - mom_mt1) / float_X(DELTA_T);
         const float_X el_factor = charge * charge
             / (float_X(6.0) * PI * EPS0 *
-               c2 * c2 * SPEED_OF_LIGHT * mass * mass);
-        const float_X energyLarmor = el_factor * gamma2 * gamma2
-            * (math::abs2(mom_dt) -
-               math::abs2(math::cross(mom, mom_dt)));
+               c2 * SPEED_OF_LIGHT * mass * mass) * gamma2 * gamma2;
```

Comment:
It's because I am using gamma^2 for bremsstrahlung, but I am also assuming orthogonal acceleration and velocity and according to the wikipedia article, this yields gamma^2 and not gamma^4 in my case. So it should be correct here.